src/pkg/oci/manifest.go

```diff
@@ -12,46 +12,42 @@ import (
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
-var (
-	// ZarfPackageIndexPath is the path to the index.json file in the OCI package.
-	ZarfPackageIndexPath = filepath.Join("images", "index.json")
-	// ZarfPackageLayoutPath is the path to the oci-layout file in the OCI package.
-	ZarfPackageLayoutPath = filepath.Join("images", "oci-layout")
-	// ZarfPackageImagesBlobsDir is the path to the directory containing the image blobs in the OCI package.
-	ZarfPackageImagesBlobsDir = filepath.Join("images", "blobs", "sha256")
-)
-
-// ZarfOCIManifest is a wrapper around the OCI manifest
+// Manifest is a wrapper around the OCI manifest
 //
 // it includes the path to the index.json, oci-layout, and image blobs.
 // as well as a few helper functions for locating layers and calculating the size of the layers.
-type ZarfOCIManifest struct {
+type Manifest struct {
 	ocispec.Manifest
 }
 
-// NewZarfOCIManifest returns a new ZarfOCIManifest.
-func NewZarfOCIManifest(manifest *ocispec.Manifest) *ZarfOCIManifest {
-	return &ZarfOCIManifest{*manifest}
+// NewManifest returns a new Manifest
+func NewManifest(manifest *ocispec.Manifest) *Manifest {
+	return &Manifest{*manifest}
 }
```

Comment:
@Noxsios Any reason we shouldn't just delete this function? It's not being used in Zarf or UDS CLI. Not sure if there's a specific reason we wanted a constructor though